ui/src/views/AutogenView.vue

```diff
@@ -1524,13 +1527,18 @@ export default {
       } else {
         callback()
       }
+    },
+    setModalWidthByScreen () {
+      const screenWidth = window.innerWidth
+      if (screenWidth <= 768) {
+        this.modalWidth = '450px'
```

Comment:
Should this be like a percentage of total width?

Reply:
No, currently I only hard fix the width for ipad, tablet screens and below. Do you feel it is ok or do you need to follow the percentage of the screen?

Reply:
I too think percentage will be better. Perhaps 90% ?